common/app/routes/Challenges/Preview.jsx

```diff
@@ -1,41 +1,17 @@
 import React, { PureComponent } from 'react';
-import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 
 import ns from './ns.json';
-import { isJSEnabledSelector } from './redux';
-import {Alert} from 'react-bootstrap';
-
 
 const mainId = 'fcc-main-frame';
 
-const mapStateToProps = state => ({
-  isJSEnabled: isJSEnabledSelector(state)
-});
+const mapStateToProps = null;
```

Comment:
If we are not using state or actions in this component, we can make it stateless and not have it connect to redux. We could go even further and it a stateless function too.